src/main/java/ua/edu/internship/interview/service/dto/SkillTreeDto.java

```diff
@@ -0,0 +1,21 @@
+package ua.edu.internship.interview.service.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.ArrayList'
<!-- __reviewdog__:ChA3NWRkYjUxZmM2MTk3YWVlEgpjaGVja3N0eWxl -->